### DISCUSSION TITLE
Cat Carrier (Corrupted Kitten) store pet is not bound to anything currently

### DISCUSSION THREAD

**Discussion by andovcr** [chromiecraft/chromiecraft#5875] [Suggestions]
This specific pet does not appear to be bound to character or account.  Could this please be investigated and fixed if necessary.  Elise mentioned: "This not being bound is an error on the shop, not an oversight, fwiw"

<img width="487" alt="image" src="https://github.com/chromiecraft/chromiecraft/assets/58115742/16f19f03-ae6f-48c8-a309-32ea0cefc77f">

**Comment by Amrx2** [comment]
Hi @andovcr, I covered this one [here](https://github.com/chromiecraft/chromiecraft/discussions/5876). Will be resolved with the next restart, thanks for flagging this to us!
